lib/_repl.scm

```diff
@@ -3535,7 +3535,7 @@
           (##write arg-num port)
           (##write-string ") " port))))
 
-  (define-prim (display-exception exc)
+  (define-prim (display-known-exception exc)
```

Comment:
so is `display-exception` now gone? that's a breaking change, can we keep it around?